cdi/pom.xml

```diff
@@ -0,0 +1,260 @@
+<project
```

Comment:
The parent pom of the cdi project should refer to the apache parent pom. See aries-rsa as an example. This should then ensure we e.g. do not need to explicitly set the apache license.